.vscode/settings.json

```diff
@@ -1,6 +1,13 @@
 {
     "files.trimTrailingWhitespace": true,
-    "[yaml]": {
+    "files.eol": "\n",
+    "[md]": {
+        "editor.wordWrap": "bounded",
+        "editor.wordWrapColumn": 75
+    },
+    "[yml]": {
+        "editor.insertSpaces": true,
         "editor.renderWhitespace": "boundary",
+        "editor.tabSize": 2
```

Comment:
This change has nothing to do with the task here and should have been done in a separate PR.